### PR TITLE
Tomw/travelers

### PR DESCRIPTION
**Pull request by Alphy11**

### CODE REVIEW

File: client/src/GamemasterInGame/TeamDistributionBar.tsx [hunk 1]
```diff
@@ -86,17 +86,17 @@ const DistributionsByPlayerCount: Record<
...
+  6: { Townsfolk: 3, Outsider: 1, Minion: 1, Demon: 1, Traveler: 0 },
+  7: { Townsfolk: 5, Outsider: 0, Minion: 1, Demon: 1, Traveler: 0 },
+  8: { Townsfolk: 5, Outsider: 1, Minion: 1, Demon: 1, Traveler: 0 },
+  9: { Townsfolk: 5, Outsider: 2, Minion: 1, Demon: 1, Traveler: 0 },
+  10: { Townsfolk: 7, Outsider: 0, Minion: 2, Demon: 1, Traveler: 0 },
+  11: { Townsfolk: 7, Outsider: 1, Minion: 2, Demon: 1, Traveler: 0 },
+  12: { Townsfolk: 7, Outsider: 2, Minion: 2, Demon: 1, Traveler: 0 },
+  13: { Townsfolk: 9, Outsider: 0, Minion: 3, Demon: 1, Traveler: 0 },
+  14: { Townsfolk: 9, Outsider: 1, Minion: 3, Demon: 1, Traveler: 0 },
+  15: { Townsfolk: 9, Outsider: 2, Minion: 3, Demon: 1, Traveler: 0 },
```
Comment (vermentor) [inline client/src/GamemasterInGame/TeamDistributionBar.tsx:99]: Weird but acceptable

File: client/src/assets/game_data/characterData.ts
Comment (vermentor) [inline client/src/assets/game_data/characterData.ts:1]: Our PRs are definitely going to conflict here, but shouldn't be too bad

File: server/src/database/gameDB.ts [hunk 2]
```diff
@@ -71,15 +75,20 @@ function createGame (): BaseUnifiedGame {
...
 export function addPlayer (
   gameId: string,
   player: string,
+  traveler?: boolean,
 ): void {
   const game = retrieveGame(gameId)
   const gameInstance = game.readOnce()
+  const gameStarted = gameInProgress(gameInstance)
+  // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+  const travelling = traveler || gameStarted
```
Comment (vermentor) [inline server/src/database/gameDB.ts:91]: Extra 'L' typo

File: server/src/database/gameDB.ts [hunk 3]
```diff
@@ -102,6 +111,10 @@ export function addPlayer (
       ...gameInstance.deadPlayers,
       [player]: false,
     },
+    travelers: {
+      ...gameInstance.travelers,
+      ...(travelling && { [player]: true }),
```
Comment (vermentor) [inline server/src/database/gameDB.ts:116]: I had no idea you could do this

**Review by vermentor — APPROVED** [review]

File: client/src/GamemasterInGame/TeamDistributionBar.tsx (hunk 1 above)
Comment (Alphy11) [reply client/src/GamemasterInGame/TeamDistributionBar.tsx:99]: yeah 🤷‍♂️

File: server/src/database/gameDB.ts (hunk 2 above)
Comment (Alphy11) [reply server/src/database/gameDB.ts:91]: fun fact, the wiki also has this typo several times

**Pull request merged**